src/failed-spec-parser.js

```diff
@@ -1,13 +1,15 @@
-const FAILED_LINE = /at \[object Object\]\.<anonymous> \((.*)\)/g;
+const FAILED_LINES = /at \[object Object\]\.<anonymous> \((.*?):.*\)/g;
 
 export default function (output = '') {
-  // this could all probably fit into one regex...
-  var failedSpecLines = output.match(FAILED_LINE);
+  var output = arguments.length <= 0 || arguments[0] === undefined ? '' : arguments[0];
 
-  if (!failedSpecLines) { return []; }
-
-  return failedSpecLines.map(function (line) {
-    let path = line.match(/\((.*):/)[1];
-    return path.slice(0, [path.length - 2]);
-  });
-}
+  var failedSpecLines = [];
+  var match;
+  //iterate over all matches and prevent adding a spec twice (e.g. when using multiCapabilities)
+  while (match = FAILED_LINES.exec(output)) {
+    if (failedSpecLines.indexOf(match[1]) == -1) {
```

Comment:
This is an awesome improvement. I wonder if this might be a good use case for a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) which would accomplish the same thing without the need for `indexOf` checking